src/megaclient.cpp

```diff
@@ -9871,6 +9871,12 @@ void MegaClient::queueread(handle h, bool p, SymmCipher* key, int64_t ctriv, m_o
     else
     {
         it->second->enqueue(offset, count, reqtag, appdata);
+        if (overquotauntil && overquotauntil > Waiter::ds)
```

Comment:
@javiserrano are you aware of this change? it's not in `develop` yet, so I wonder how it ended up here in this branch

Reply:
Yes, it's to fix the management of the overquota state on streaming transfers.
The problem was reported by Android developers and the fix has been already confirmed to be working fine.